mappings/net/minecraft/entity/EntityType2.mapping

```diff
@@ -0,0 +1,50 @@
+CLASS net/minecraft/class_3460 net/minecraft/entity/EntityType2
+	COMMENT Really just a EntityType, but there already is a mapping file for that
```

Comment:
Maybe instead of EntityType2, we can figure out what it's for, and work from there?

Reply:
Or this could be an intermediary problem, in which case we should get it matched so it's all EntityType anyway.

Reply:
I don't want to break intermediary compatibility again :pensive:

Reply:
It'd be one class, it'd probably break very little.

Reply:
that's not "how it works"

Reply:
I did EntityType2 -> EntityTypes, I think it's fine now